bridge/slack/slack.go

```diff
@@ -385,11 +389,16 @@ func (b *Bslack) uploadFile(msg *config.Message, channelID string) {
 		ts := time.Now()
 		b.Log.Debugf("Adding file %s to cache at %s with timestamp", fi.Name, ts.String())
 		b.cache.Add("filename"+fi.Name, ts)
+		initialComment := fmt.Sprintf("File from %s", msg.Username)
+		if fi.Comment != "" {
+			initialComment += fmt.Sprintf("with comment: %s", fi.Comment)
```

Comment:
There's not going to be a space between the existing message and the addition. I'd suggest:
```suggestion
			initialComment += fmt.Sprintf("%s with comment: %s", initialComment, fi.Comment)
```

Reply:
Your suggestion gives:
![image](https://user-images.githubusercontent.com/1810977/48983036-4080da80-f0ea-11e8-8fca-6590bd989243.png)

How it currently works for me:
![image](https://user-images.githubusercontent.com/1810977/48983097-da488780-f0ea-11e8-901c-e23a1bb2982e.png)

The space should normally already be there because you should've set in the `RemoteNickFormat` :) (which should be visible if you run with `PrefixMessagesWithNick=true`)

I don't agree to adding extra spaces here

Reply:
`RemoteNickFormat` should not end with a space as it is not necessary when you are rewriting the posting-pseudo.

We should in general not be relying on users specifying such essentials things themselves. The input should be sanitised by removing spaces at front and end and then we should add spaces where we need them in the Slack bridge itself.

Reply:
I understand your pov, but if we do this we'll probably get requests of users that don't want a default space after the nick.
The reasoning was to explicitly not set spaces so users have full control about how the username is shown by modifying the `RemoteNickFormat`.